src/SDK/Language/CLI.php

```diff
@@ -0,0 +1,357 @@
+<?php
+
+namespace Appwrite\SDK\Language;
+
+use Appwrite\SDK\Language;
+
+class CLI extends Language {
```

Comment:
You can extend the PHP class and remove a lot of methods from this class

Reply:
Perfect 👍